meetings/notes/sig-meeting-20230221.md

```diff
@@ -0,0 +1,49 @@
+## Meeting Details
+
+- **Date/Time:** [Feb 21 2023 @ 17:00pm UTC / 11:00am ET / 9:00am PT](https://lists.o3de.org/g/o3de-calendar/viewevent?repeatid=39350&eventid=1573469&calstart=2023-02-21)
+- **Location:** [Discord SIG-Network Voice Room](https://discord.gg/62nq7HP5mP)
+- **Moderator:** Pip Potter (lmbr-pip)
+- **Note Taker** Pip Potter (lmbr-pip)
+
+[Agenda Issue](https://github.com/o3de/sig-network/issues/92)
+
+## SIG Updates
+
+### General SIG announcements
+* SIG elections held, current chair (lmbr-pip) and co-chair (kberg-amzn) re-elected
+* Wanted to call out community members Gene (Gene \[Amazon\]) and Olex (Olex \[Amazon\]) for all their help on Discord with spawning and server launching issues, in sig-network text channel.
+
+### Multiplayer Sample updates
+![Screenshot](../media/2023/mps_feb_21.png)
+
+* New lighting pass
+* Lots of character updates
+* Fixes to client/server stability
+* Focus is ensuring PvP is responsive and functional.
+* Followup with Royal about build/access as he's interested in early access.
+* Expect to contribute new sample in March to o3de/o3de-multiplayersample
+
+### Networking Updates
+* Added network component icons and missing tooltips
+* Fixes for array/vector serialization
+* Lots of cvar, cfg and setreg changes
+* Added new player id debug component
+
+### Cloud Services
```

Comment:
feels strange to have this section w/o any content. Suggest either a bullet which says "N/A" or mentions the MP Scaler update from below

Reply:
Will add